LEAF_Request_Portal/templates/view_reports.tpl

```diff
@@ -888,6 +888,37 @@ var indicatorSort = {}; // object = indicatorID : sortID
 var grid;
 let gridColorData = {}; //object updated with id: color
 let tempColorData = {}; //object updated with id: color
+let isOneFormType = false;
+
+/**
+ * Purpose: Check if only one type of form could logically be returned and,
+ * if so, update global variables isOneFormType (bool) and categoryID (string).
+ * @param searchQueryTerms - variable with result of leafSearch.getLeafFormQuery().getQuery().terms
+ */
+function checkIfOneTypeSearchedAndUpdate(searchQueryTerms) {
```

Comment:
Need some type checking on searchQueryTerms.  I know IE doesn't like it in the call, but inside the function we need to ensure it's actually and object so later calls don't fail badly.